c31-shadowlyn/react-nutshell-shadowlyn
# Rw add new task

# Description

This pull request displays the tasks for the user when they click on the tasks button. A button to add a new task is displayed on the DOM. A form appears when that button is clicked allowing the person to enter in a new task. When they click submit, that new task is posted to the database and rendered on the DOM.

Applies to issue #13 + #14 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout rw-addNewTask`
3. run npm start
4. cd into api and run `json-server -p 5002 -w database.json`
5. go to localhost:5002
6. click on tasks
7. do the tasks with your user id appear on the DOM? cool!
8. click add new task
9. enter in new task and submit
10. is your new task on the DOM and in the database? even cooler!
11. pat yourself on the back you deserve it.


# Checklist:
